android-sdk/tasks/main.yml

```diff
@@ -50,7 +50,7 @@
 
 -
   name: "Get the Android SDK pod name"
-  shell: "oc get pods --namespace={{ project_name }} | grep android-sdk | awk '{print$1}'"
+  shell: "oc get pods --namespace={{ project_name }} | grep android-sdk | grep -v \"-deploy\" | awk '{print$1}'"
```

Comment:
We can also add a failed_when: output.stdout == ""
I.e. if there is no android pod available we should stop the playbook here because it will fail at the point of trying to install the Android SDK if there is no value in android_sdk_pod